bblfsh/compat.py

```diff
@@ -0,0 +1,228 @@
+import os
```

Comment:
Can you please add a comment to this file summarizing what it is and outlining the API?

Even if the answer is just "this is the v1 API surface (see link) and bridges to a v2 client", that would be helpful for the future reader.

Reply:
Will do.